src/features/filters/server/filterToPrisma.ts

```diff
@@ -58,7 +58,7 @@ export function filterToPrismaSql(
         break;
       case "number":
       case "numberObject":
-        valuePrisma = Prisma.sql`${filter.value}::DOUBLE PRECISION`;
+        valuePrisma = Prisma.sql`${filter.value.toFixed(6)}::DOUBLE PRECISION`;
```

Comment:
Was this here the fix to make the query work or was this related to any fof the changes to the input field?

Reply:
This fix was meant to fix the query error, we sent a message about it on Slack [here](https://gitstart.slack.com/archives/C066S52EC3D/p1706709032520979)